src/PrestaShopBundle/Controller/Admin/Sell/Order/ActionsBarButtonsCollection.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Controller\Admin\Sell\Order;
+
+class ActionsBarButtonsCollection
+{
+    /**
+     * @var ActionsBarButton[]
+     */
+    protected $buttons = [];
+
+    /**
+     * @return bool
+     */
+    public function isEmpty()
+    {
+        return empty($this->buttons);
+    }
+
+    /**
+     * @param ActionsBarButton $button
+     */
+    public function addButton(ActionsBarButton $button)
```

Comment:
Must be an interface, same for the collection
```suggestion
    public function addButton(ActionsBarButtonInterface $button)
```

Reply:
I thought about it, but wouldn't it be a little overkill ? It's just some button with 3 properties ... almost like a VO. We dont put interfaces on VO, do we ?